WordPress/src/main/java/org/wordpress/android/ui/reader/subfilter/viewholders/SubfilterListItemViewHolder.kt

```diff
@@ -23,9 +24,13 @@ open class SubfilterListItemViewHolder(
         }
         val itemText: TextView? = this.itemView.findViewById(R.id.item_title)
         itemText?.let {
-            if (filter.isSelected) {
-                it.setTextColor(ContextCompat.getColor(parent.context, R.color.primary))
-            }
+            it.setTextColor(ContextCompat.getColor(
+                    parent.context,
+                    if (filter.isSelected)
```

Comment:
This is super minor, but can we add curly brackets to if/else here? It could prevent some silly issues in the future :)

Reply:
👍changed here f97bbf90c05ca39e67825776a94c71d15142de1c